### PR TITLE
corrected manual gateway instance creation flow

### PR DESCRIPTION
**Pull request by inflatador**
Incorrect info about subnet update, replaced with correct info about network and subnet creation.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-10cf90bb6c/docs/user-guides/infrastructure/)

File: cloud-config/network/cloud-networks-product-concepts/servicenet-publicnet-requirement.rst [hunk 1]
```diff
@@ -45,10 +45,17 @@ How to set up Internet access for servers without PublicNet or ServiceNet
   configuration at the same time.
 
 
-1) Create a Cloud Network via the portal.
-:rax-dev-docs:`Update the Cloud Network subnet
-<cloud-networks/v2/api-reference/subnet-operations/#update-subnet>`
-to include the following information:
+1) :rax-dev-docs:`Create a Cloud Network via
```
Comment (stephamon) [inline cloud-config/network/cloud-networks-product-concepts/servicenet-publicnet-requirement.rst]: I'd rephrase as:

"Create a Cloud Network by using the neutron API or the neutron CLI. You cannot create the network by using the Cloud Control Panel."
